packages/taquito-sapling/src/sapling-state/interface.ts

```diff
@@ -0,0 +1,23 @@
+export interface SaplingStateDiff {
+  root: string;
+  commitments_and_ciphertexts: [string, SaplingCipherText][];
+  nullifiers: string[];
+}
+
+export interface SaplingCipherText {
```

Comment:
This type is already present in the RPC package and could be imported from there.